core/views.py

```diff
@@ -15,13 +15,13 @@ def __init__(self):
         super(HashedNameServiceClient, self).__init__()
 
     def name_to_hash(self, name):
-        return requests.get(f"http://35.178.109.215:5000/name2hash/{urllib.parse.quote(name)}").text
+        return requests.get(f"https://cbp-id-provider.herokuapp.com/name2hash/{urllib.parse.quote(name)}").text
 
     def hash_to_name(arg, hash):
-        return urllib.parse.unquote(requests.get(f"http://35.178.109.215:5000/hash2name/{hash}").text)
+        return urllib.parse.unquote(requests.get(f"https://cbp-id-provider.herokuapp.com/hash2name/{hash}").text)
 
     def hash_to_info(arg, hash):
-        return urllib.parse.unquote(requests.get(f"http://35.178.109.215:5000/hash2info/{hash}").text)
+        return urllib.parse.unquote(requests.get(f"https://cbp-id-provider.herokuapp.com/hash2info/{hash}").text)
```

Comment:
Same comment as above for `hash2name`, this isn't working for me using the hash I received from the example input name.

Reply:
@RyanBalfanz same comment as above, try with the links:

 - [adzhPj8Ck3xCnrguGhdO](https://cbp-id-provider.herokuapp.com/hash2info/adzhPj8Ck3xCnrguGhdO)
 - [cX6Vm8qcsqpDQJtzFcU7](https://cbp-id-provider.herokuapp.com/hash2info/cX6Vm8qcsqpDQJtzFcU7)
 - [iRKG4uYxKRWuBa6psK22](https://cbp-id-provider.herokuapp.com/hash2info/iRKG4uYxKRWuBa6psK22)
 - [KeS3H4FFWfB4bdZnliev](https://cbp-id-provider.herokuapp.com/hash2info/KeS3H4FFWfB4bdZnliev)